Specialkost - test till daniel

AlexanderGrip opened the pull request:
Tog väldigt lång tid att göra. Koden är med all säkerhet fel någonstans. Du får väldigt gärna kolla igenom den om du vill Jag förstår inte varför texten. Blir stor i bland. :( Sen glömde jag duplicatea skriftlig varning! men jag duplicateade den som jag hade skrivit i och då den inte var sparad (tror jag) så fick jag den tillbaka :)

AlexanderGrip committed:
Började med anhållan om lov

AlexanderGrip committed:
Specialkost - test

Specialkost - test

AlexanderGrip committed:
Annhållan om lov från skolan

Var det såhär du ville ha den?
Är den här godkänd annars tycker du ? :)

solidfox commented:
Du verkar inte ha syncat specialkost blanketten innan du gjorde en pull request.

solidfox commented:
Syncat

solidfox commented:
Eller så har du bara inte commitat

AlexanderGrip committed:
Blankett för nyanställda

Tycker du att den ser okej ut? :)

AlexanderGrip commented:
Specialkosten testade jag att göra hemifrån.
Jag testar synka den igen sen hemma! jag gjorde en ny blankett för ny
anställda! och sparade den som html format för att jag ville testa och se
hur den blev!
tyvärr blev den uhm :(
Sådär!

2017-01-19 10:22 GMT+01:00 Daniel Schlaug <notifications@github.com>:

> Eller så har du bara inte commitat
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#issuecomment-273721545>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AYB721gGls2trZsSpKYiJZBaakvLe9kqks5rTytggaJpZM4LnC_h>
> .
>

solidfox commented:
Det går inte att visa dem som separata HTML dokument. Alla filer i de två mapparna måste också finnas på plats, annars blir det konstigt.

All the best,
Daniel Schlaug

Machine Learning Master Student
@ KTH Royal Institute of Technology

2017年1月19日 10:33 +0100 に AlexanderGrip <notifications@github.com> は書きました:
> Specialkosten testade jag att göra hemifrån.
> Jag testar synka den igen sen hemma! jag gjorde en ny blankett för ny
> anställda! och sparade den som html format för att jag ville testa och se
> hur den blev!
> tyvärr blev den uhm :(
> Sådär!
>
> 2017-01-19 10:22 GMT+01:00 Daniel Schlaug <notifications@github.com>:
>
> > Eller så har du bara inte commitat
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#issuecomment-273721545>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AYB721gGls2trZsSpKYiJZBaakvLe9kqks5rTytggaJpZM4LnC_h>
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub (https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#issuecomment-273723903), or mute the thread (https://github.com/notifications/unsubscribe-auth/AAS4dvoGyS6vw5jcYhbdHR-RkeSFbXvTks5rTy3NgaJpZM4LnC_h).
>
>
>

AlexanderGrip commented:
Åh. så jag måste "synka" "Blankett för nyanställda med mapparna css, images
och js?

2017-01-19 10:35 GMT+01:00 Daniel Schlaug <notifications@github.com>:

> Det går inte att visa dem som separata HTML dokument. Alla filer i de två
> mapparna måste också finnas på plats, annars blir det konstigt.
>
> All the best,
> Daniel Schlaug
>
> Machine Learning Master Student
> @ KTH Royal Institute of Technology
>
> 2017年1月19日 10:33 +0100 に AlexanderGrip <notifications@github.com> は書きました:
> > Specialkosten testade jag att göra hemifrån.
> > Jag testar synka den igen sen hemma! jag gjorde en ny blankett för ny
> > anställda! och sparade den som html format för att jag ville testa och se
> > hur den blev!
> > tyvärr blev den uhm :(
> > Sådär!
> >
> > 2017-01-19 10:22 GMT+01:00 Daniel Schlaug <notifications@github.com>:
> >
> > > Eller så har du bara inte commitat
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#
> issuecomment-273721545>,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> AYB721gGls2trZsSpKYiJZBaakvLe9kqks5rTytggaJpZM4LnC_h>
> > > .
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub (
> https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#
> issuecomment-273723903), or mute the thread (https://github.com/
> notifications/unsubscribe-auth/AAS4dvoGyS6vw5jcYhbdHR-
> RkeSFbXvTks5rTy3NgaJpZM4LnC_h).
>
> >
> >
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#issuecomment-273724481>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AYB72w9RajhMiF2TKsdWZzWvbKgv0Dehks5rTy5vgaJpZM4LnC_h>
> .
>

AlexanderGrip committed:
Anhållan om lov - underskrift och telefonnummer delning

Anhållan om lov - underskrift och telefonnummer delning

AlexanderGrip committed:
Medveten motorisk träning

el klar. ska skruva på en dator skärm :)

AlexanderGrip committed:
Merge branch 'master' of https://github.com/AlexanderGrip/Gripsholmsskolan-Forms

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+
+			<table>
+			  <tr>
+					<tr
```
Ta bort <tr

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+			<table>
+			  <tr>
+					<tr
+					<label>Uppgifter om Eleven</label>
```
Uppgifter om Eleven ska vara en h2, inte en label.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+					<tr
+					<label>Uppgifter om Eleven</label>
+			      <td colspan=2>
+							<label>För- och efternamn</label>
```
Dela upp i två celler. En för förnamn och en för efternamn.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+						<label>Årskurs</label>
+						<textarea rows=1></textarea>
+					</td>
+			</tr>
```
Indentera! (Lägg på en tabb innan.)

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+						<label>Vårdnadshavare 1</label>
+						<textarea rows=1></textarea>
+					</td>
+					<td id="socialsecuritynumber">
```
Ta bort id.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+			</tr>
+
+			<tr>
+				<td id="name">
```
Ta bort id

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+					<label>Mobiltelefon</label>
+					<textarea rows=1></textarea>
+				</td>
+				<td id="socialsecuritynumber">
```
Ta bort id

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+	</table>
+
+	<table>
+			<tr
```
Ta bort <tr

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+
+	<table>
+			<tr
+			<label>Eleven är allergisk mot</label>
```
Detta ska vara en h2, inte en label.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+		</td>
+
+		<td id="socialsecuritynumber">
+<input type="checkbox" name="vehicle" value="Bike">
```
Fixa så det ser snyggt ut med indenteringen (tabbarna)

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+</label>
+		</td>
+
+		<tr>
```
Avsluta förra raden med </tr> och ta bort en tab på  denna.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+		</td>
+		<td colspan=2>
+			<input type="checkbox" name="vehicle" value="Bike">
+		<label>Annat:</label>
```
Ta bort checkboxen helt och hållet.

solidfox replied:
Och ta bort : från Annat:

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+<tr>
+		<td colspan=4>
+			<label>Vad gör vi när eleven får en allergisk reaktion? </label>
+			<textarea rows=1></textarea>
```
Du måste avsluta `<td>` här.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+<tr>
+		<td colspan=4>
+			<label>Ska eleven använda medicin vid en eventuell reaktion?</label>
+			<form action="">
```
Använd inte `<form>` (i dagsläget)

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+			<tr
+			<label>Eleven är allergisk mot</label>
+		<td id="name">
+			<input type="checkbox" name="vehicle" value="Bike">
```
Ta bort name="vehicle" value="Bike" på alla checkboxes.

solidfox commented on Specialkost.html:
```diff
@@ -0,0 +1,220 @@
+		<label>Annat:</label>
+		<textarea rows=1></textarea>
+		</td>
+</tr>
```
Indentering. Härifrån och neråt.

solidfox reviewed (changes requested):
La in lite kommentarer, tror du kan ändra på en massa andra ställen med samma typ av fel när du fixat dessa. Du kan också ta bort alla id="???". De bara stökar till det.

AlexanderGrip committed:
Utredning av en elevs behov av särskilt stöd

Utredning av en elevs behov av särskilt stöd

Mycket kodfel.
Men om du vill så får du jätte gärna kolla igenom den.

AlexanderGrip committed:
Dom viktiga dokumenten

Dom viktiga dokumenten
Jag tror att jag har gjort. Alla. Jag ska kolla igenom dom mer senare

AlexanderGrip committed:
Medveten Motorisk Träning

Nu är den klar så länge :)

AlexanderGrip committed:
Svenska som Andraspråk

Svenska som Andraspråk

AlexanderGrip committed:
Uppsägning Av Elevplats På Gripsholmsskolan

Uppsägning Av Elevplats På Gripsholmsskolan

AlexanderGrip committed:
Uppsägning Av Elevplats På Gripsholmsskolan

små fixad :)

AlexanderGrip committed:
Lilla Klassen

Lilla Klassen

AlexanderGrip committed:
Daniel fixar bugg som jag råkade göra med koden :(

Daniel fixar bugg som jag råkade göra med koden :(

AlexanderGrip committed:
Alla ändrade

AlexanderGrip committed:
ny

AlexanderGrip committed:
WIP - Dokumentation av extra listor

håller på med listor (^-^ )/
<ul>
<li>
</li>
</ul>

AlexanderGrip committed:
mera ändringar

AlexanderGrip committed:
mer ändringar :)

AlexanderGrip committed:
:)

AlexanderGrip committed:
vet inte vad jag ska göra med betygsättningen

inväntar daniel om han har några idéer tills nästa gång

AlexanderGrip committed:
mera

AlexanderGrip committed:
mera :)

AlexanderGrip committed:
ändringar

solidfox commented:
"Mera" är inte det bästa att skriva på sin commit. Försök beskriva vad du gjort. T ex "Lagt till `<label>` taggar." eller "Ändrat `<br>` taggar till `<li>` taggar."

AlexanderGrip commented:
Det kan vara smart! Tack Daniel!
ska försöka komma i håg detta och tänka på! :)

ha en fin kväll :)

Den 16 februari 2017 14:12 skrev Daniel Schlaug <notifications@github.com>:

> "Mera" är inte det bästa att skriva på sin commit. Försök beskriva vad du
> gjort. T ex "Lagt till taggar." eller "Ändrat
> taggar till
> - taggar."
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/solidfox/Gripsholmsskolan-Forms/pull/1#issuecomment-280326484>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AYB72_j73Ehb0q70jubdiWkp2bzQR-HYks5rdEsmgaJpZM4LnC_h>
> .
>

AlexanderGrip committed:
Hemuppgift målformulering åk 6–7 Skapad

Hemuppgift målformulering åk 6–7 Skapad

AlexanderGrip committed:
Stödlektioner - Dokumentation skapad

Stödlektioner - Dokumentation skapad

AlexanderGrip committed:
Anmälningsblankett sommarskola skapad.

Anmälningsblankett sommarskola skapad.
